core/src/main/java/org/polypheny/db/algebra/constant/SemiJoinType.java

```diff
@@ -77,16 +76,16 @@ public JoinAlgType toJoinType() {
     }
 
 
-    public CorrelateJoinType toLinq4j() {
+    public JoinType toLinq4j() {
         switch ( this ) {
             case INNER:
-                return CorrelateJoinType.INNER;
+                return JoinType.INNER;
             case LEFT:
-                return CorrelateJoinType.LEFT;
+                return JoinType.LEFT;
             case SEMI:
-                return CorrelateJoinType.SEMI;
+                return JoinType.LEFT_SEMI_JOIN;
             case ANTI:
-                return CorrelateJoinType.ANTI;
+                return JoinType.COMMA;
```

Comment:
Isn't a comma join the same thing as a cartesian join (cross join)? This is not the same as an anti join.

Reply:
This is a renaming from the external library.